### DISCUSSION TITLE
Convert file download directly from .ts to .mp4

### DISCUSSION THREAD

**Discussion by KryCool** [filecxx/FileCentipede#701] [Ideas]
Almost the videos file download currently being .ts video, so I have idea about the convert directly from .ts to .mp4 when the file  is completed

**Comment by filecxx** [comment]
It will be supported in the future.

**Comment by carl467** [comment]
@filecxx I understand that this feature is going to be added in the future, but would it also be possible to have the option to do direct stream copy to mp4 or other format (i.e. like stream copy using ffmpeg) and to be able select the option under the stream section in settings?

**Reply by filecxx** [reply]
Yes.

**Comment by carl467** [comment]
@filecxx Thank you for response
